server/client.go

```diff
@@ -1497,7 +1497,15 @@ func (c *client) processConnect(arg []byte) error {
 	lang := c.opts.Lang
 	account := c.opts.Account
 	accountNew := c.opts.AccountNew
+
+	// If JWT is not in opts, imply cookie JWT
```

Comment:
Since this really applies only for websocket clients, I would have done it this way:
```
// If websocket client and JWT not in the CONNECT, use the cookie JWT (possibly empty).
if ws := c.ws; ws != nil && c.opts.JWT == "" {
    c.opts.JWT = ws.cookieJwt
}
```
What do you think?